ocfweb/tv/templates/tv/hours.html

```diff
@@ -1,7 +1,5 @@
-{% extends 'tv/base.html' %}
 {% load tv_formatting %}
```

Comment:
I don't think you need this extra load here if you have it in the parent template, but I could be wrong on that.

Reply:
Unfortunately, parents do not pass these onto their children :(